Sources/Plasma/PubUtilLib/plNetClient/plNetLinkingMgr.cpp

```diff
@@ -112,6 +120,8 @@ struct NlmLeaveAgeOp : NlmOp {
     NlmLeaveAgeOp ()
     : NlmOp(kNlmOpLeaveAgeOp), quitting(false)
     { }
+
+    virtual ~NlmLeaveAgeOp() = default;
```

Comment:
This doesn't need to be declared -- the base is already virtual, and this override is a no-op.